databricks-sdk-java/src/main/java/com/databricks/sdk/core/retry/StatementExecutionRetryStrategyPicker.java

```diff
@@ -0,0 +1,31 @@
+package com.databricks.sdk.core.retry;
+
+import com.databricks.sdk.core.http.Request;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class StatementExecutionRetryStrategyPicker implements RetryStrategyPicker {
```

Comment:
Can we have better name for this class? It has nothing to do with StatementExecution. Something like ListBasedRetryStrategyPicker or RetryStrategyPickerImpl etc